camaragon/rancid-tomatillos
# Ch/feature/cypress testing

#### Please review considerations below:


#### What's this PR do? 

Integrates Cypress testing to the current project's file paths.  Some functionality was refactored to satisfy the test results.  The layout on the single movie display page was also refactored for user readability. 

#### Where should the reviewer start? 

In the project directory, open the <Cypress> folder ---> the <integration> folder and open movie_display.spec.js and movie_info.spec.js in your script editor to view the code. 

#### How should this be manually tested? 

Open Cypress to run the testing suites with `npm cypress open` and view test results to see them pass. 

#### Any background context you want to provide? 

It was hard to tell whether fetch requests were working in the tests and why they were or were not grabbing the elements we wanted them to, so it took some refactoring.  We also incorporated instructor notes per our last code review to refactor some functionality.
